libevmasm/Assembly.cpp

```diff
@@ -73,6 +80,205 @@ unsigned Assembly::codeSize(unsigned subTagSize) const
 	}
 }
 
+void Assembly::importAssemblyItemsFromJSON(Json::Value const& _code, std::vector<std::string> const& _sourceList)
+{
+	solAssert(m_items.empty());
+	solRequire(_code.isArray(), AssemblyImportException, "Supplied JSON is not an array.");
```

Comment:
This isn't covered by any tests from what I can tell (same goes for line 103 - not an object).

Reply:
Yeah, most of those validations aren't. I'd prefer them covered, but that would require either tons of CLI tests, a boost-based test or a new testing mechanism for this. And Daniel really wanted us to get this merged without fussing too much about details since this is just an experimental feature. At this point what matters to me is that all the important cases of the import are covered, and they seem to be at this point.